src/main/java/org/jitsi/videobridge/rest/root/Application.java

```diff
@@ -14,19 +14,25 @@
  * limitations under the License.
  */
 
-package org.jitsi.videobridge.rest.shutdown;
+package org.jitsi.videobridge.rest.root;
 
 import org.glassfish.jersey.server.*;
 import org.jitsi.utils.logging2.*;
-import org.jitsi.videobridge.util.*;
+import org.jitsi.videobridge.rest.binders.*;
+import org.jitsi.videobridge.rest.filters.*;
+import org.osgi.framework.*;
 
-public class ShutdownApp extends ResourceConfig
+public class Application extends ResourceConfig
```

Comment:
NOTE that github got confused here.  `ShutdownApp` was removed entirely and `Application` is new